pkg/record/diskrecorder/diskrecorder.go

```diff
@@ -245,6 +245,9 @@ func (r *Recorder) PeriodicallyPrune(ctx context.Context, reported AlreadyReport
 				}
 				return true, nil
 			})
+			if err != nil {
+				klog.V(2).Infof("Failed to wait for exponential backoff : %v", err)
```

Comment:
This would deserve better message IMO. Is it only info level?

Reply:
@tremes any suggestion about the message text? How about changing it to `klog.Errorf`?